### ISSUE TITLE
Cannot read property 'loc' of undefined (v1.1.0)

### ISSUE THREAD

**Issue by jarretmoses** [HeroProtagonist/eslint-plugin-module-resolver#136]
Hello, I upgraded to v1.1.0 and I started getting the error in the title `Cannot read property 'loc' of undefined`. I haven't touched anything in my config besides updating the package. Digging into it a little I isolated the issue to my rule set in `'module-resolver/use-alias': 2` which was working as expected in `1.0.0`.

**Comment by HeroProtagonist** [comment]
Thanks. `1.1.0` includes a fix that was supposed to address https://github.com/HeroProtagonist/eslint-plugin-module-resolver/issues/44, which sounds like the same issue. 
If you want to take a look into this issue feel free. I should have time early next week. Either way sounds like for now you can go back to `1.0.0`

**Comment by vith** [comment]
I hit the same issue just now, and I can confirm downgrading from `1.1.0` to `1.0.0` solved it.

**Comment by HeroProtagonist** [comment]
Would either of you be able to share what your config file for babel looks like?

**Comment by dipsaus9** [comment]
I ran into the same issue with 1.0.0 and 1.1.0.
It looks like the same issue as #44 indeed but I can not figure out why.

On version 1.0.0 I have the following babel config:
```js
// .babelrc.js
 module.exports = {
	presets: ["next/babel"],
	plugins: [
		'babel-plugin-emotion',
		['module-resolver',
		{
			alias: {
				'~': './src',
			},
		}],
	],
};
```

This results into the following eslint error: `Unable to find config for babel-plugin-module-resolver eslint(module-resolver/alias)`

With version 1.1.0 I get the `cannot read property loc of undefined` error.

When I update my `.babelrc.js` file to a `babel.config.json` file on version 1.0.0 I also get the `cannot read property loc of undefined` error.

And the other way around. When I use version 1.1.0 and I update my `.babelrc.js` to a `babel.config.json` I get the `Unable to find config for babel-plugin-module-resolver eslint(module-resolver/alias)` error. 

I hope this will help solve the issue.

**Comment by jarretmoses** [comment]
@HeroProtagonist sorry I have been busy. I can try and take a look over the weekend.

**Comment by HeroProtagonist** [comment]
👋 Would any of you be able to see if the code in `1.2.0-beta` fixes it? If it does I will add a few things to the PR and then deploy `1.2.0`

Thanks and happy holidays 🎉

**Comment by jarretmoses** [comment]
@HeroProtagonist ill check tomorrow

**Comment by jarretmoses** [comment]
@HeroProtagonist looks like this is indeed fixed in 1.2.0-beta

**Comment by HeroProtagonist** [comment]
Please update to version `1.2.0`. This should fix the issue. If anything else arises please open another issue. 

Happy New Year! 🎆
